cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/SsaImplCommon.qll

```diff
@@ -287,20 +287,6 @@ private module SsaDefReaches {
     )
   }
 
-  /**
-   * Holds if the SSA definition of `v` at `def` reaches uncertain SSA definition
-   * `redef` in the same basic block, without crossing another SSA definition of `v`.
-   */
-  predicate ssaDefReachesUncertainDefWithinBlock(
```

Comment:
Does this change preserve identical files?

Reply:
It does because all the identical files were modified :)

Reply:
Yes, the code is dead in all copies.